README.md

```diff
@@ -121,6 +121,30 @@ docker-compose up
 (Note that on MacOS you may have to run `TMPDIR=/private$TMPDIR docker-compose up` if
 `$TMPDIR` contains a symbolic link that cannot be mounted by Docker.)
 
+Use on existing cocker-compose project. Add in existing services. The project can be found in docker hub, no need to download or clone source:
```

Comment:
@whummer Whoops, careful on the spelling of "docker-compose". :joy:

Reply:
outch! good catch, thanks